content/docs/2.3/scalers/azure-event-hub.md

```diff
@@ -27,10 +27,19 @@ triggers:
 
 - `connectionFromEnv` - Name of the environment variable your deployment uses to get the connection string appended with `EntityPath=<event_hub_name>`
 - `storageConnectionFromEnv` - Name of the environment variable that provides connection string for Azure Storage Account to store checkpoint. As of now the Event Hub scaler only reads from Azure Blob Storage.
-- `consumerGroup` - Consumer group of event hub consumer. (default: `$default`)
+- `consumerGroup` - Consumer group of Event Hub consumer. (default: `$default`)
 - `unprocessedEventThreshold` - : Average target value to trigger scaling actions. (default: 64)
-- `blobContainer` - Container name to store checkpoint. This is needed when a using an Event Hub application written in dotnet or java, and not an Azure function
+- `blobContainer` - Container name to store checkpoint. This is needed for every `checkpointStrategy` except of `AzureFunction`. With Azure Functions the `blobContainer` is autogenerated and cannot be overridden.
+- `checkpointStrategy` - configure the checkpoint behaviour of different Event Hub SDK's. (default: `""`)
+    - `""`: if no checkpointStrategy is specified, the Event Hub scaler is in backward compatibility mode and able to scale older implementations of C#, Python or Java Event Hub SDK's. (see "Legacy checkpointing"). if this behaviour should be used, `blobContainer` is also required.
+    - `azureFunction`: suitable for Azure Functions. This is automatically the default setting, if `blobcontainer` is empty.
+    - `blobMetadata`: For all implementations which stores checkpoint informations on blob metadata like current C#, Python, Java and JavaScript Event Hub SDK's.
+    - `goSdk` For all implementations which use the Golang SDK checkpointing like Dapr.io
 
+**Legacy checkpointing**  
+C# applications, which use the `Microsoft.Azure.EventHubs` package, Java applications which use the `azure-eventhubs-eph` package  or Python applications which use the `azure-eventhub` below `v5` are supported if no `checkpointStrategy`is specified. All these legacy implementations based on the `EventProcessorHost` Client, which stores checkpoint information as blobcontainer content
+
+C# Applications which use the current `Azure.Messaging.EventHubs` package, Java applications which use the current `azure-messaging-eventhubs` package or Python applications since `azure-eventhub v5`, have to set the `checkpointStrategy`to `blobMetadata`. All these implementations based on the `EventProcessorClient` which stores checkpoint information as blobcontainer metadata.
```

Comment:
Let's use a quote to align with other scalers
```suggestion
> 💡 **Legacy Checkpointing:** C# applications, which use the `Microsoft.Azure.EventHubs` package, Java applications which use the `azure-eventhubs-eph` package or Python applications which use the `azure-eventhub` below `v5` are supported if no `checkpointStrategy` is specified. These legacy implementations are based on the `EventProcessorHost` client, which stores checkpoint information as blob container content
> 
> C# Applications that use the current `Azure.Messaging.EventHubs` package, Java applications which use the current `azure-messaging-eventhubs` package or Python applications since `azure-eventhub v5`, have to set the `checkpointStrategy`to `blobMetadata`. All these implementations based on the `EventProcessorClient` which stores checkpoint information as blob metadata.

```